pwa-devdocs/src/technologies/basic-concepts/internationalization/index.md

```diff
@@ -0,0 +1,157 @@
+---
+title: Internationalization Framework
+---
+
+Internationalization (i18n) is a feature that lets you localize content for a culture, region, or language.
+This feature is often associated with localization (l10n), which is the process of transforming content for a specific locale.
+
+## Internationalization in core Magento versus PWA Studio
+
+The core Magento application includes an i18n feature that provides translated text to the frontend theme.
+This feature uses dictionary files inside language packages to provide translation data for Magento when it renders a page.
+The language packages themselves are Magento extensions the application installs using Composer.
+
+For more information, see the Magento core topic: [Translations overview][].
+
+The tight coupling between core Magento's i18n feature and its frontend theme makes it difficult to use the same translation mechanisms in PWA Studio storefronts.
+Instead, PWA Studio provides its own i18n framework that follows a similar design as the one in core Magento.
+
+## How it works
+
+PWA Studio provides a context provider for translations called the [`LocaleProvider`][].
+This context provider contains translation data from dictionary files and supplies them to its child components.
+
+The [`react-intl`][] library powers the i18n framework in PWA Studio.
+It provides the [`IntlProvider`][] component to PWA Studio's `LocaleProvider`, which configures the component to use the storefront's translation data.
+This library also provides [`formatMessage()`][] and [`FormattedMessage`][] to localize text in child components.
+
+The following code samples produce the same results:
+
+```jsx
+return (
+    <p className={classes.text}>
+        <FormattedMessage id={LOREM_IPSUM} />
```

Comment:
`LOREM_IPSUM` doesn't follow our ID format which we mention later on, let's be consistent and use an ID which matches our best practices, `component.loremIpsum` for instance.

Reply:
updated